PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterizer.h

```diff
@@ -69,6 +69,8 @@ class AliEmcalCorrectionClusterizer : public AliEmcalCorrectionComponent {
   Bool_t                 fJustUnfold;                     ///< just unfold, do not recluster
   Float_t                fUnfoldCellMinE;                 ///< min energy cell threshold, after unfolding
   Float_t                fUnfoldCellMinEFrac;             ///< min fraction of cell energy after unfolding  
+  Int_t                  fNxMRowDiff;                     ///< NxN clusterizer, max number rows from center cell: 1->3, 2->5, 3->7
+  Int_t                  fNxMColDiff;                     ///> NxN clusterizer, max number columns from center cell:1->3, 2->5, 3->7
```

Comment:
Typo in the streaming comment - `>` -> `<`

Reply:
Do you think this is considered so far from the ///?
Anyway, I will change it.

Reply:
`///<` is meaningful, but `///>` isn't. The rest of the `>` in the comment are totally fine